src/domains/pendingProjectAuthorization/repository/pendingProjectAuthorization.repository.impl.ts

```diff
@@ -1,19 +1,48 @@
 import { type PendingProjectAuthorizationRepository } from '@domains/pendingProjectAuthorization/repository/pendingProjectAuthorization.repository';
-import { PendingProjectAuthorizationDTO, type PendingProjectInputDTO } from '@domains/pendingProjectAuthorization/dto';
+import { PendingProjectAuthorizationDTO } from '@domains/pendingProjectAuthorization/dto';
 import type { PendingProjectAuthorization, PrismaClient } from '@prisma/client';
 import type { ITXClientDenyList } from '@prisma/client/runtime/library';
+import { type AuthorizationRequest } from '@domains/integration/dto';
+import { encryptData } from '@utils';
 
 export class PendingProjectAuthorizationRepositoryImpl implements PendingProjectAuthorizationRepository {
   constructor(private readonly db: PrismaClient | Omit<PrismaClient, ITXClientDenyList>) {}
 
-  async create(data: PendingProjectInputDTO): Promise<PendingProjectAuthorizationDTO> {
```

Comment:
Logic enhanced, congrats! I was thinking that if we are using transactional approach with different db entities it shouldn't be in PendingProject repo. As we recieve data in Integration controller we should create an integration repository which can handle transactions among multiple tables. It is cleaner than having them gattered in different places